packages/wrangler/src/__tests__/delete.test.ts

```diff
@@ -155,22 +157,40 @@ describe("delete", () => {
 		mockListKVNamespacesRequest(...kvNamespaces);
 		// it should only try to delete the site namespace associated with this worker
 
-		setMockResponse(
-			"/accounts/:accountId/storage/kv/namespaces/id-for-my-script-site-ns",
-			"DELETE",
-			([_url, accountId]) => {
-				expect(accountId).toEqual("some-account-id");
-				return null;
-			}
+		msw.use(
+			rest.delete(
+				"*/accounts/:accountId/storage/kv/namespaces/id-for-my-script-site-ns",
+				(req, res, ctx) => {
+					expect(req.params.accountId).toEqual("some-account-id");
+					return res.once(
+						ctx.status(200),
+						ctx.json({
+							success: true,
+							errors: [],
+							messages: [],
+							result: {},
+						})
+					);
+				}
+			)
 		);
 
-		setMockResponse(
-			"/accounts/:accountId/storage/kv/namespaces/id-for-my-script-site-preview-ns",
-			"DELETE",
-			([_url, accountId]) => {
-				expect(accountId).toEqual("some-account-id");
-				return null;
-			}
+		msw.use(
```

Comment:
This msw mock seems like it's used a lot, can it be extracted out into a function? Something like `mockDeleteSites()`?

Reply:
Rewrite is currently the focus. Cleanup and abstractions are welcome as follow up PRs